blog/BCHN-Financial-Report-2020-10-20.md

```diff
@@ -0,0 +1,218 @@
+
+Currently, the project has received and is holding donations of 2,000,000 SPICE 🌶️ and 1,000 USDt on that address.
+
+A few other unknown tokens have been sent to the address.
+
+As we conduct our payments exclusively in Bitcoin Cash up to now, we have yet to set up a ledger file to account for token holdings. We may do so for main tokens like SPICE, but not generally for all random tokens sent our way, as this would be a way to spam the project.
+
+
+### Development bounties
+
+The project has had good experience with bountied work so far and has increased that share of its expenditure.
+
+In this last period, bounties were used to successfully source
+
+* website content (importing from read.cash to the Newsroom)
+
+* development of a relatively complex new functional test related to ongoing performance improvements
+
+* development of a new `indexdir` feature (complete with implementation, development and documentation) requested by a user
+
+Developers who are interested in partaking in upcoming bounties, are invited to join the project Slack (link below) for announcements in the development channels and keep an eye out for [Issues on GitLab marked with a `bounty` label](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=bounty).
+
+* * *
+
+### Links
+
+Previous financial overviews:
+
+*   [Financial overview in report of 14 March 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-03-14-84f2992d#financial-overview)
+
+*   [Financial overview in report of 31 March 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-03-31-1e0a4c39#financial-overview)
+
+*   [Financial overview in report of 15 April 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-04-15-449fc115#financial-overview)
+
+*   [BCHN Financial Report 2020-06-03](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-06-03-d2a0232f) (3 June 2020)
+
+*   [BCHN Financial Report 2020-07-07](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-07-07-d04c7e65) (7 July 2020)
+
+*   [BCHN Financial Report 2020-09-03](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-09-03-a477d4e2) (20 October 2020)
+
+Flipstarter funding proposal:
+
+*   [English, v1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/raw/master/finance/funding/campaigns/2020-q2-flipstarter/pdf/Bitcoin_Cash_Node_Flipstarter_Funding_Proposal_v1_1.pdf) (PDF)
+
+*   [Chinese, v1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/blob/master/finance/funding/campaigns/2020-q2-flipstarter/pdf/Bitcoin_Cash_Node_Flipstarter_Funding_Proposal_v1_1_CN.pdf) (PDF)
+
+*   [Spanish, 1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/blob/master/finance/funding/campaigns/2020-q2-flipstarter/proposal/proposal-es.mediawiki) (mediawiki)
+
+**Getting in touch with BCHN:**
```

Comment:
This additional formatting makes it look inconsistent in the final rendering.

![image](https://user-images.githubusercontent.com/34025062/96753219-80ea4f80-140a-11eb-8423-fdc3553284c9.png)

Reply:
Sorry the "website" whiteout is my bad and unrelated

Reply:
Fixed in #139